back-end/interfaces/IQuestionnaireQuestion.ts

```diff
@@ -2,6 +2,7 @@ export interface IQuestionnaireQuestion {
   id: string;
   text: string;
   quizText: string;
+  tags: string[];
```

Comment:
store an array of `tags` on each question - used to determine which questions belong to which sets